userland/examples/ble_adv/main.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include "ble.h"
+#include <timer.h>
+
+int main(void)
+{
+  printf("\r\ndemo app\n");
+  unsigned char name[] = "TockOS";
+  unsigned char data[] = "1337";
+  unsigned char tx[] = {0x1};
+
+  printf("return for ble_adv_set_power %d\r\n", ble_adv_set_txpower(0));
+  printf("return for ble_adv_set_interval %d\r\n", ble_adv_set_interval(0));
```

Comment:
At the moment it is hard-coded to 150ms and the syscall has no impact. 
However we will implement it in the next refactor and the capsule will validate the interval accordingly.